web/src/components/pages/dashboard/stats/leaderboard-card.tsx

```diff
@@ -126,9 +126,10 @@ class UnconnectedLeaderboard extends React.Component<Props, State> {
     );
   }
 
-  playAvatarClip = function(clipUrl: string, position: any) {
+  playAvatarClip = function(clipUrl: string, position: any, self: boolean) {
     const { locale } = this.props;
-    trackVoiceAvatar('listen', locale);
+    if (self) trackVoiceAvatar('self-listen', locale);
+    else trackVoiceAvatar('listen', locale);
```

Comment:
personally I'd go [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here, e.g. `trackVoiceAvatar(self ? 'self-listen' : 'listen', locale)`, but I'm happy to merge it in either way. Thanks! :)